app/3.0/toolbox/localize.md

```diff
@@ -4,113 +4,123 @@ title: Localization
 **you must load a polyfill yourself.** An example polyfill can be found at
 [github.com/andyearnshaw/Intl.js](https://github.com/andyearnshaw/Intl.js/).
 
-`Polymer.AppLocalizeBehavior` supports the same
+`AppLocalizeBehavior` supports the same
 [message syntax](http://formatjs.io/guides/message-syntax/) as `format.js`, in its entirety; use the
 library docs as reference for the available message formats and options.
 
-Each element that displays content to be localized should add `Polymer.AppLocalizeBehavior`.
-All of these elements share a common localization cache, so you only need to load translations once.
+## Install AppLocalizeBehavior
 
-`AppLocalizeBehavior` can be used directly in Polymer 1.x elements and Polymer 2.x hybrid elements.
-For class-style elements, use the `mixinBehaviors` method.
+    npm install --save @polymer/app-localize-behavior@next
 
-## Install AppLocalizeBehavior
+## Import AppLocalizeBehavior
+
+AppLocalizeBehavior is a legacy behavior. To use it in Polymer 3.0, you need the utility class `mixinBehaviors` as well:
+
+Import AppLocalizeBehavior {.caption}
 
-    bower install --save PolymerElements/app-localize-behavior
+```js
+// Import the PolymerElement base class
+import {PolymerElement, html} from '@polymer/polymer/polymer-element.js';
 
+// Import AppLocalizeBehavior 
+import {AppLocalizeBehavior} from '@polymer/app-localize-behavior/app-localize-behavior.js';
+
+// Import a utility class to treat legacy behaviors as mixins
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+
+// Define your class, inheriting AppLocalizeBehavior functions and properties
+class AppLocalize extends mixinBehaviors([AppLocalizeBehavior], PolymerElement) {
+```
+  
 ## Add localization to your app
 
 The main application is usually responsible for loading the localized messages and setting the
 current language.
 
-Sample application (class style elements) {.caption}
-
-```html
-<dom-module id="x-app">
-  <template>
-    <!-- use the localize method to localize text -->
-    <div>{{localize('hello', 'name', 'Batman')}}</div>
-  </template>
-  <script>
-    class XApp extends Polymer.mixinBehaviors([Polymer.AppLocalizeBehavior], Polymer.Element) {
-      static get is() { return 'x-app'}
-
-      static get properties() {
-        return {
-          // set the current language—shared across all elements in the app
-          // that use AppLocalizeBehavior
-          language: {
-            value: 'en'
-          },
-
-          // Initialize locale data
-          resources: {
-            value() {
-              return {
-                'en': { 'hello': 'My name is {name}.' },
-                'fr': { 'hello': 'Je m\'apelle {name}.' }
-              }
-            }
-          }
-        };
+app-localize.js {.caption}
+
+```js
+// Imports go here
+class AppLocalize extends mixinBehaviors([AppLocalizeBehavior], PolymerElement) {
+  static get properties(){
+    return{
```

Comment:
space after `return`

Or alternately, it would be more idiomatic and more readable to set these values in the constructor and leave off the `propeties` object entirely:

```js
constructor() { 
  super();
  // set default language and localization strings
  this.language = 'en';
  this.resources = {
    'en': { 'hello': 'My name is {name}.' },
    'fr': { 'hello': 'Je m\'apelle {name}.' }
  };
}